Sources/TokamakLint/Models/TokamakLint.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  TokamakLint.swift
+//  Tokamak
+//
+//  Created by Matvii Hodovaniuk on 4/1/19.
+//
+
+import Foundation
+import SwiftSyntax
+
+public func lintFolder(_ path: String) throws {
+//  let resourceKeys: [URLResourceKey] = [.creationDateKey, .isDirectoryKey]
+//  let baseurl = URL(fileURLWithPath: path)
+//  guard let enumerator = FileManager
+//    .default
+//    .enumerator(at: baseurl,
+//                includingPropertiesForKeys: resourceKeys,
+//                options: [.skipsHiddenFiles],
+//                errorHandler: { (url, error) -> Bool in
+//                  print("directoryEnumerator error at \(url): ", error)
+//                  return true
+//    })?.compactMap({ $0 as? URL }).filter({ isSwiftFile($0.path) })
+//  else {
+//    fatalError("Enumerator is nil")
+//  }
+//  let count = enumerator.count
+//  let enumerated = enumerator.enumerated()
+//  for (i, fileURL) in enumerated {
+//    print("Linting ",
+//          "\(fileURL.lastPathComponent) ",
+//          "(\(i)/\(count))")
+//    let visitor = try walkParsedTree(fileURL.path)
+//    if hasTokamakImport(from: visitor) {
+//      guard try isPropsEquatable(fileURL.path) else {
+//        throw LintError.propsIsNotEquatable
+//      }
+//    }
+//  }
+}
+
+public func lintFile(_ path: String) throws -> [StyleViolation] {
+  let visitor = try walkParsedTree(path)
+  guard !hasTokamakImport(from: visitor) else {
+    return []
+  }
+  var errors: [StyleViolation] = []
+  errors.append(contentsOf: PropsIsEquatableRule.validate(visitor: visitor))
+  return errors
```

Comment:
this looks like it doesn't need `errors` intermediate variable and can return with a plain `return PropsIsEquatableRule.validate(visitor: visitor)`